core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -83,6 +84,11 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
   def AdminAddLibraryCurator(userEmail: RawlsUserEmail) = asFCAdmin { addLibraryCurator(userEmail) }
   def AdminRemoveLibraryCurator(userEmail: RawlsUserEmail) = asFCAdmin { removeLibraryCurator(userEmail) }
 
+  val dmPubSubTopic = dmConfig.getString("pubSubTopic")
```

Comment:
I would create a `UserServiceConfig` case class, and initialize it in `boot`...if anything's wrong with config, we want to know sooner than later. Something like https://github.com/broadinstitute/sam/blob/develop/src/main/scala/org/broadinstitute/dsde/workbench/sam/Boot.scala#L61